.gitignore

```diff
@@ -3,3 +3,4 @@ dist/
 *.egg-info/
 qabot/__pycache__/
 qabot/lib/__pycache__
+tests/__pycache__/
```

Comment:
I think we can just have one line `__pycache__`, though this is not a blocker for this PR